source/codegen/metadata/nidmm/functions.py

```diff
@@ -1345,302 +1798,373 @@
     },
     'Init': {
         'cname': 'niDMM_init',
+        'codegen_method': 'public',
         'init_method': True,
         'parameters': [
             {
+                'cppName': 'resourceName',
                 'direction': 'in',
+                'grpc_type': 'string',
                 'name': 'resourceName',
-                'type': 'ViRsrc'
+                'type': 'ViString'
             },
             {
+                'cppName': 'idQuery',
                 'direction': 'in',
+                'grpc_type': 'bool',
                 'name': 'idQuery',
                 'type': 'ViBoolean'
             },
             {
+                'cppName': 'resetDevice',
                 'direction': 'in',
+                'grpc_type': 'bool',
                 'name': 'resetDevice',
                 'type': 'ViBoolean'
             },
             {
+                'cppName': 'vi',
                 'direction': 'out',
+                'grpc_type': 'nidevice_grpc.Session',
                 'name': 'vi',
                 'type': 'ViSession'
             },
             {
+                'cppName': 'errorMessage',
                 'direction': 'out',
-                'get_last_error': 'get_last_error_message',
+                'get_last_error': 'deprecated',
```

Comment:
The value for 'get_last_error' keys don't matter anymore, just need the param here so it gets generated in the proto as deprecated.